packages/client/hmi-client/src/services/data.ts

```diff
@@ -10,6 +11,7 @@ import {
 	XDD_RESULT_DEFAULT_PAGE_SIZE
 } from '../types/XDD';
 
+const HMI_SERVER_BASE_API_URL = 'http://localhost:8078/api';
```

Comment:
Why are you not able to just use `/api` here?

Reply:
good catch. code is updated